go.mod

```diff
@@ -120,6 +120,7 @@ require (
 	golang.org/x/crypto v0.0.0-20190510104115-cbcb75029529
 	golang.org/x/net v0.0.0-20190812203447-cdfb69ac37fc
 	golang.org/x/oauth2 v0.0.0-20190402181905-9f3314589c9a
+	golang.org/x/tools v0.0.0-20191124021906-f5828fc9a103 // indirect
```

Comment:
how exactly was this dependency added to go.mod? does it need to be added to tools.go? I did something similar in #7952 but if this is sufficient then i can rebase mine down to just the Makefile update.